app/views/fae/application/_content_form.html.slim

```diff
@@ -1,5 +1,6 @@
 ruby:
   require_locals     ['attribute', 'f'], local_assigns
+  blank_label          = !label || label == '' unless label.nil?
   label              ||= attribute.to_s.titleize
```

Comment:
@andrewdang17 We're starting to spread the label logic all over this file. The `label_adjusted` assignment on line 13 is where the full label get's built, so I think that's where the conditional logic should live. There should be no need to execute that logic if we're just setting it to false later.

Reply:
I agree I just did it this way because I didn't want to add more `if/unless` statements to lines 12-22 or have to wrap the whole thing in an if statement